src/skylink/sia.ts

```diff
@@ -53,15 +53,56 @@ export class SiaSkylink {
     // Remove padding characters.
     return trimSuffix(base64, "=");
   }
+
+  /**
+   * Loads the given raw data and returns the result. Based on sl.LoadBytes in
+   * skyd.
+   *
+   * @param data - The raw bytes to load.
+   * @returns - The sia skylink.
+   * @throws - Will throw if the data has an unexpected size.
+   */
+  static loadBytes(data: Uint8Array): SiaSkylink {
+    // Sanity check the size of the given data.
+    if (data.length !== RAW_SKYLINK_SIZE) {
+      throw new Error("Failed to load skylink data");
+    }
+
+    const buf = data.buffer;
+    const view = new DataView(buf);
```

Comment:
nit: We don't gain anything from copying the data here.
```suggestion
const view = new DataView(data.buffer);
```